pkg/chaosdaemon/util.go

```diff
@@ -40,6 +41,7 @@ const (
 // ContainerRuntimeInfoClient represents a struct which can give you information about container runtime
 type ContainerRuntimeInfoClient interface {
 	GetPidFromContainerID(ctx context.Context, containerID string) (uint32, error)
+	ContainerKillByContainerID(ctx context.Context, containerID string) error
```

Comment:
Actually I am still not sure about the naming. I think both `GetPidFromContainerID` and `ContainerKillByContainerID` is quite redundant. It is okay to just use `GetPid` and `KillContainer`. WDYT @fewdan  @zhouqiang-cl

Reply:
I'm not sure if there are other ways in the future, such as not based on "ID" but on "image" and the like. So I think we can name it longer now.

Reply:
Sure. That makes sense 😄